backend/api/Controllers/Models/CustomMissionQuery.cs

```diff
@@ -7,8 +7,8 @@ public struct CustomInspectionQuery
         public InspectionType InspectionType { get; set; }
 
         public float? VideoDuration { get; set; }
+        public AnalysisType AnalysisType { get; set; }
```

Comment:
Should this be required? I think optional would make more sense

Reply:
Should be optional as other missions - done now n